specification/maps/data-plane/Search/stable/2023-06-01/search.json

```diff
@@ -0,0 +1,1202 @@
+      "get": {
+        "description": "**Geocoding**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nIn many cases, the complete search service might be too much, for instance if you are only interested in traditional geocoding. Search can also be accessed for address look up exclusively. The geocoding is performed by hitting the geocoding endpoint with just the address or partial address in question. The geocoding search index will be queried for everything above the street level data. No Point of Interest (POIs) will be returned. Note that the geocoder is very tolerant of typos and incomplete addresses. It will also handle everything from exact street addresses or street or intersections as well as higher level geographies such as city centers, counties, states etc.",
+        "operationId": "Search_GetGeocoding",
+        "produces": [
+          "application/geo+json"
+        ],
+        "x-ms-examples": {
+          "Search detail address 15127 NE 24th Street, Redmond, WA": {
+            "$ref": "./examples/ForwardGeocoding.json"
+          },
+          "Search detail address 15127 NE 24th Street, Redmond, WA by addressLine": {
+            "$ref": "./examples/ForwardGeocodingByAddressLine.json"
+          },
+          "Search detail address 15127 NE 24th Street, Redmond, WA by query": {
+            "$ref": "./examples/ForwardGeocodingByQuery.json"
+          },
+          "Search landmark Empire State Building by query": {
+            "$ref": "./examples/ForwardGeocodingFindLandmarkByQuery.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "$ref": "#/parameters/Top"
+          },
+          {
+            "name": "query",
+            "in": "query",
+            "description": "A string that contains information about a location, such as an address or landmark name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "addressLine",
+            "in": "query",
+            "description": "The official street line of an address relative to the area, as specified by the locality, or postalCode, properties. Typical use of this element would be to provide a street address or any official address.\n\n**If query is given, should not use this parameter.**",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "countryRegion",
+            "in": "query",
+            "description": "Restrict the geocoding result to an [ISO 3166-1 Alpha-2 region/country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) that is specified e.g. FR. This will limit the search to the specified region.\n\n**If query is given, should not use this parameter.**",
+            "required": false,
+            "type": "array",
```

Comment:
@jsedlak-microsoft I was looking at the request from double checking everything. Peter noted that `countryRegion` is changing from `array` to `string` because we're filtering only the first country/region (matching the [code changes](https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-IoT-LocationBasedServices/pullrequest/8829268?path=/src/ServiceGateway/ServiceGateway/Transforms/GeocodingRequestV2.cs&_a=files).